### ISSUE TITLE
Namespace required for installing prompt

### ISSUE THREAD

**Issue by ryzokuken** [matchai/spacefish#17]
Here's what happen when I try to install it in the normal way (as suggested by docs):

```
❯ fisher spacefish
Installing 1 plugin/s
! Fetch spacefish github.com/fisherman/spacefish
! There was an error installing spacefish or more plugin/s.
Try using a namespace before the plugin name: owner/spacefish
```

Easy fix:

```
~/Code/banter master 
❯ fisher matchai/spacefish
Installing 1 plugin/s
OK Fetch spacefish github.com/matchai/spacefish
Done in 15s 984ms
```

After this, it just proceeds to throw #16

**Comment by matchai** [comment]
Hey Ujjwal! Thank you for opening an issue.

You shouldn't need to provide a namespace when installing spacefish, since it has been added to the fisher index here: https://github.com/fisherman/index/blob/master/index#L487-L491

I would suggest trying to run the following commands:
```
__fisher_remote_index_update
fisher update
fisher install spacefish
```

I reinstalled spacefish on my end and didn't encounter any issues.
Let me know if it still fails to install 😄
